src/helpers/fixtures.js

```diff
@@ -60,7 +60,13 @@ let fixtureHelper = module.exports = {
         }
       }
 
-      if (options.query && !_.isEqual(options.query, fixture.request.query)) {
+      const filteredFixturedQuery = xhrHelper.getQueryStringObject({
+        url: 'http://localhost/?' + Object.keys(fixture.request.query || {}).map(function(key) {
```

Comment:
If not that then something indicating that the domain isn't important might make a better replacement for `localhost`.